src/odemis/acq/move.py

```diff
@@ -502,6 +620,253 @@ def constrain_stage_pos_axes(self, stage_val: Dict[str, float], fixed_sample_axe
         else:
             raise NotImplementedError(f"Posture {posture} not supported")
 
+    ###### DEPRECATE START #######
+    # These functions can be removed once 3D transforms are fully implemented
```

Comment:
Let's then complete the 3D transforms.

Reply:
This will be removed alongside the feature flags